src/Ztm.WebApi.Tests/SqlCallbackRepositoryTests.cs

```diff
@@ -0,0 +1,162 @@
+using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore;
+using Xunit;
+using Ztm.Data.Entity.Contexts;
+using Ztm.Data.Entity.Contexts.Main;
+using Ztm.Data.Entity.Testing;
+
+namespace Ztm.WebApi.Tests
+{
+    public class SqlCallbackRepositoryTests
+    {
+        readonly ICallbackRepository subject;
```

Comment:
We don't need to use interface due to this class is the tests for `SqlCallbackRepository`. So just use concrete class directly is better.

Reply:
make sense